vmm/src/gdb.rs

```diff
@@ -76,6 +128,303 @@ pub trait Debuggable: vm_migration::Pausable {
         data: &[u8],
     ) -> std::result::Result<(), DebuggableError>;
     fn active_vcpus(&self) -> usize;
+    fn coredump(&mut self, _destination_url: &str) -> std::result::Result<(), DebuggableError> {
+        Ok(())
+    }
+    fn coredump_init(&mut self, _destination_url: &str) -> std::result::Result<DumpState, DebuggableError> {
+        Ok(DumpState::new(0, 0, 0, 0, None, None))
+    }
+    fn coredump_begin(&mut self, _dump_state: &DumpState) -> std::result::Result<(), DebuggableError> {
+        Ok(())
+    }
+    fn coredump_iterate(&mut self, _dump_state: &DumpState) -> std::result::Result<(), DebuggableError> {
+        Ok(())
+    }
+    fn get_mem_offset(&self, _phdr_num: u16, _note_size: isize) -> u64 {
+        0
+    }
+}
+
+/// transfer anything to &[u8]
+pub unsafe fn any_as_u8_slice<T: Sized>(p: &T) -> &[u8] {
```

Comment:
I think we already have `ByteValued` that already does this?

Reply:
ByteValued works find. Thanks.